src/main/java/mods/eln/misc/Utils.java

```diff
@@ -875,17 +853,16 @@ public static Object getItemObject(ItemStack stack) {
         return i;
     }
 
-    @Deprecated
     static public void getItemStack(String name, List list) {
         // TODO: Fuck this function.
         Iterator itItem = Item.REGISTRY.iterator();
-        NonNullList<ItemStack> tempList = NonNullList.create();
```

Comment:
There was probably a reason for this being a non-null list. Making an arbitrary array list with an arbitrary size isn't a solution I don't think.